### PR TITLE
Add /community/users-list

### PR DESCRIPTION
**Pull request by didib**
Changes proposed in this pull request:

- Add /community/users-list

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @didib 

This pull request needs review by: @sandrobonazzola

### CODE REVIEW

File: source/community/users-list.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+# Users list
+
+The [users mailing list](https://lists.ovirt.org/archives/list/users@ovirt.org/) is the main place for getting help, asking questions, consulting about design/hardware/configuration/performance etc., and of course sharing your experience.
+
+## Getting help
+
+TL;DR: When asking for help, please provide: what you did, versions of packages used, content of `/var/log/*ovirt*` and `/var/log/vdsm/*` . Thanks!
+
```
Comment (sandrobonazzola) [inline source/community/users-list.md:15]: Also `yum -q list installed centos-release ovirt-release\* ovirt-engine redhat-release vdsm glusterfs` as per https://lists.ovirt.org/archives/list/users@ovirt.org/thread/GPCHNEZU56NVWBPFEOR6OF5KQF3T5NGS/

Comment (didib) [reply source/community/users-list.md:15]: Added this below. Above line is "TL; DR" - I don't want to make it any longer, perhaps better shorten if you can think of something...

File: source/community/users-list.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Users list
+authors: didib
```
Comment (sandrobonazzola) [inline source/community/users-list.md:3]: On ovirt site your user is `didi`

Comment (didib) [reply source/community/users-list.md]: Thanks

**Comment by didib** [comment]
I think it's ready for merge, no? Removed the do-not-merge label.

**Review by sandrobonazzola — APPROVED** [review]

**Pull request merged**